lib/create-command-server.js

```diff
@@ -70,27 +58,56 @@ const onConnection = websocket => {
   });
 };
 
+class CommandServer extends WebSocketServer {
+  /**
+   * Broadcast message to all clients.
+   * @param message
+   */
+  broadcast(message) {
+    const packedMessage = JSON.stringify(message);
+    /** @type {Set<WebSocketWithData>} */ (this.clients).forEach(websocket => {
+      if (websocket.sessionId) {
+        websocket.send(packedMessage, error => {
+          if (error) {
+            this.emit('error', `error sending message: ${error}`);
+          }
+        });
+      }
+    });
+  }
+}
+
 /**
  * Create a server which communicates with external clients using the WebSocket
  * protocol described in the project README.md file.
  *
  * @param {number} port - the port on which the server should listen for new
  *                        connections
  *
- * @returns {Promise<EventEmitter>} an eventual value which is fulfilled when
- *                                  the server has successfully bound to the
- *                                  requested port
+ * @returns {Promise<CommandServer>} an eventual value which is fulfilled when
+ *                                   the server has successfully bound to the
+ *                                   requested port
  */
 module.exports = async function createWebSocketServer(port) {
-  const server = new WebSocketServer({
+  const server = new CommandServer({
     clientTracking: true,
     path: '/session',
     port,
   });
   await new Promise(resolve => server.once('listening', resolve));
 
-  server.broadcast = broadcast.bind(null, server);
```

Comment:
Adding a key not defined on the type (in this case `WebSocketServer`) produces an error. So I extended `WebSocketServer` as `CommandServer` and added the method to it.